src/style.css

```
@@ -1 +1,118 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src:
+  local("Avenir"),
+  url(fonts/Avenir.ttc),
+  url(fonts/Avenir-Book.ttf),
+  url(fonts/Avenir-Heavy.ttf);
+  font-size: 300px;
+}
+
+form {
+  width: 100%;
+  margin-top: 20px;
+}
+
+.big-search {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  border-radius: 3px;
+  border: 1px solid #E1E7ED;
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  align-items: center;
+}
+
+
+.big-search:hover{
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-img {
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  position: absolute;
+  }
+
+  .search-img.big {
+    width: 19px;
+    height: 19px;
+    margin-left: 26px;
+  }
+
+.big-search input {
+  display: flex;
+  font-size: 16px;
+  border: none;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  padding-left: 62px;
+
+}
+
+.small-search {
+  display: flex;
+  width: 100%;
+  height: 42px;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+}
+
+.small-search:hover{
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-img.small {
+  width: 11px;
+  height: 11px;
+  margin-left: 13px;
+  background-size: contain;
+}
+
+.small-search input {
+  border: none;
+  font-size: 14px;
+  margin-left: 9px;
+  width: 100%;
+  height: 100%;
+  padding-left: 33px;
+}
+
+.small-search input::placeholder{
+  color: #3D4E61;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 350;
+  line-height: normal;
+}
+
+.big-search input::placeholder {
+  display: flex;
+  color: #3D4E61;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+}
+
+
+input:focus {
+  outline: none;
+  border-radius: 3px;
+  border: 1px solid #E1E7ED;
+  background: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+  color: #3D4E61;
+  font-style: normal;
+  font-weight: 800;
+  line-height: normal;
```

Default values, so you can just remove it everywhere
```suggestion
  font-weight: 800;
```